### PR TITLE
create index.html basic structure

### PR DESCRIPTION
**Pull request by mattlub**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
```
Comment (MohamedOmarii) [inline index.html]: Why this page not have a title !!?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <nav>
+      <ul>
+        <li><a href="#top">Home</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#team">Team</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section>
```
Comment (MohamedOmarii) [inline index.html]: I think the name of this html element could be a `header` not a 
`section`

File: index.html (hunk 1 above)
Comment (mattlub) [inline index.html]: I will let you put a title in your branch

File: index.html (hunk 2 above)
Comment (mattlub) [inline index.html]: We thought this was slightly better semantics because it is the introduction section, not just a header.

Comment (mattlub) [inline index.html]: But we will change it to header.

Comment (Jbarget) [inline index.html]: This describes what @MohamedOmarii is talking about well :) 
https://www.w3schools.com/tags/tag_header.asp

Comment (MohamedOmarii) [inline index.html]: I think it should to be in `header` because it is in a top of the page

File: index.html (hunk 1 above)
Comment (MohamedOmarii) [inline index.html]: Ok , good job

File: index.html (hunk 2 above)
Comment (MohamedOmarii) [inline index.html]: Good job guys !

**Pull request merged**